cdc/kv/region_worker.go

```diff
@@ -193,9 +193,9 @@ func (w *regionWorker) handleSingleRegionError(err error, state *regionFeedState
 		zap.String("namespace", w.session.client.changefeed.Namespace),
 		zap.String("changefeed", w.session.client.changefeed.ID),
 		zap.Uint64("regionID", regionID),
-		zap.Uint64("requestID", state.getRequestID()),
-		zap.Stringer("span", state.getRegionSpan()),
-		zap.Uint64("resolvedTs", state.getRegionInfoResolvedTs()),
+		zap.Uint64("requestID", state.requestID),
+		zap.Stringer("span", state.sri.span),
+		zap.Uint64("resolvedTs", state.sri.resolvedTs),
```

Comment:
How can we guarantee there is not data race with `state.sri.resolvedTs` after removing lock.

Reply:
the value of the `state.sri` never be changed after the state is created?